edudoc/competition-templates/competition-templates.md

```diff
@@ -4,59 +4,66 @@
 DO NOT use any templates before reading this entire document!
 :::::
 
-Here you can find printable templates for cube covers and Clock covers, both of which are stackable; 3D-printable cover designs for Square-1, Clock and Skewb puzzles; results boxes for scorecards: a design that can be hung on a wall and another that can be put on a table; Square-1 inserts (21 per page) and an outline of some important competition rules.
+Here you can find the following printable templates that can be used for WCA competitions:
+
+- Cube covers and Clock covers, both stackable
+- 3D-printable cover designs for Square-1, Clock, and Skewb puzzles
+- Results boxes for scorecards: one that can be put on a table and another that can be hung on a wall
+- Square-1 inserts (21 per page)
+- Competitor guide infographic with illustrations of the most important rules
+- Multi-Blind scrambling infographic, with illustrations that show best practices
+- Sheets with cube outlines for tracking Multi-Blind scrambles (14 cubes per page)
 
 ![](images/results_boxes_and_cube_cover.jpg){.centered width=600px}
 
 ## Instructions
 
-1. If you would like to use the WCA designs, use the files in the folder for the language used in your region. **WARNING: Some of the PDF previews might have rendering artifacts, but these won't show up if you download the files.** If you would like to make custom designs, use the files in the `Editable Files` folder. **DO NOT** make colored Clock cover designs, they must be in black and white (BW).
-2. The designs for paper Clock covers and competition rules can be printed on a home printer; the Clock covers need to be printed in BW and the competition rules need to be printed in color. The other designs **MUST** be printed using an industry-grade printer, so you will have to find a printing place in your area. They will require `PDF` files to make your prints.
+1. If you would like to use the WCA designs, use the files in the folder for the language used in your region, if available. If you would like to make custom designs, use the files in the `Editable Files` folder. **DO NOT** make Clock cover designs that use a lot of ink, because that makes the paper more rigid, which can affect Clock pins. It is preferable to have a minimal design in Black and White (BW).
```

Comment:
```suggestion
1. If you would like to use the WCA designs, use the files in the folder for the language used in your region, if available. If you would like to make custom designs, use the files in the `Editable Files` folder.
```
This change would make sense if the Clock pins proposal is approved by the Board.

Reply:
We should do that in a separate PR when that happens